src/containers/ResourcesPage/index.jsx

```diff
@@ -6,39 +6,34 @@ import resourcesForEducation from "../../images/resourcesForEducation.png";
 import whoCausesClimate from "../../images/whoCausesClimate.png";
 import { useTranslation } from "react-i18next";
 
-const initResources = [
-  {
-    id: 1,
-    title: "Our changing climate - UNICEF",
-    body:
-      "This book is a child friendly handbook meant to explain the concepts of climate change in an easily understandable manner and language. It aims to link behavioural choices to the changes observed in our climate",
-    pic: ourClimateChange,
-    url:
-      "gs://capstone-istanbul-juniper.appspot.com/ResourcesPage/Child friendly climate change handbook.pdf",
-  },
-  {
-    id: 2,
-    title: "Resources for Education on Climate Change - UNESCO",
-    body:
-      "This publication focuses on the challenges, opportunities and good practices of climate change. It explores the interrelationship between climate change and lifestyles through a scientific, political, economic, social, ethical and cultural angle and identifies actions young people might take towards more sustainable lifestyles. It provides information, case studies and useful tips around topics relevant to young people and their everyday lives, such as food and drink, travel and transport, leisure and entertainment. The guide was produced for young people and people working with them such as educators, teachers, trainers and youth leaders around the world",
-    pic: resourcesForEducation,
-    url:
-      "gs://capstone-istanbul-juniper.appspot.com/ResourcesPage/Resources list_Climate Change.pdf",
-  },
-  {
-    id: 3,
-    title: "Who Causes Climate changes - TROCAIRE",
-    body:
-      "Children explore who contribute to climate change by looking at activities related to climate change using climate change bingo sheet, creating clouds that list activates causing gas emissions and represent the world population by dividing themself.",
-    pic: whoCausesClimate,
-    url:
-      "gs://capstone-istanbul-juniper.appspot.com/ResourcesPage/creating-futures-lesson-4.pdf",
-  },
-];
-
 const ResourcesPage = () => {
   const [t] = useTranslation();
-
+  const initResources = [
+    {
+      id: 1,
+      title: t("resourcesPage.resourceOneTitle"),
```

Comment:
resourceOne is not better than title1

they also have the same structure, thats why an array makes sense

your two options are to make the code either look like this:

t("resourcesPage.resources[0].title")
t("resourcesPage.resources[0].text")
(or something like this, i cant remember the syntax for arrays)

or like this:
t("resourcesPage.resources.climateChangeHandbook.text") (or some other meaningful name)

both of these are cleaner, the second provides more meaning if the descriptions actually have to do with the resource themselves. which it does.

imagine your teammate needs to change this code, and they need to change the order of the arrays. and they need to decide ,"hey should i use resourceOneText or resourceTwoText for climate change handbook.pdf?" 

* more readable: t("resourcesPage.resources.climateChangeHandbook.title")
* not readable: (".resourceOneText")

Reply:
in this way, it's much much much clearer :D, Thanks. PTAL.